tests/unit/ethBridgeAddresses.test.ts

```diff
@@ -0,0 +1,47 @@
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { expect } from 'chai'
+import {
+  EthBridge,
+  getEthBridgeInformation,
+  getL2Network,
+} from '../../src/lib/dataEntities/networks'
+import dotenv from 'dotenv'
+dotenv.config()
+
+/**
+ * Tests the getEthBridgeInformation function against the information
+ * of Arbitrum One network (42161)
+ */
+describe('Obtain deployed bridge addresses', () => {
+  it('obtains deployed ETH Bridge addresses', async () => {
```

Comment:
I'd move this test to integration tests since we're actually hitting the network and we're not mocking anything